packages/dna_tools.py

```diff
@@ -0,0 +1,150 @@
+from typing import Union
```

Comment:
```suggestion
from typing import Union


```

По PEP8 импорты от основного кода надо отделять двумя пустями строками